src/commands/synthetics/__tests__/run-test.test.ts

```diff
@@ -254,7 +254,12 @@ describe('run-test', () => {
       jest.spyOn(runTests, 'getApiHelper').mockImplementation(() => apiHelper as any)
       await expect(
         runTests.executeTests(mockReporter, {...ciConfig, publicIds: ['public-id-1', 'public-id-2'], tunnel: true})
-      ).rejects.toMatchError(new CriticalError('TRIGGER_TESTS_FAILED'))
+      ).rejects.toMatchError(
+        new CriticalError(
+          'TRIGGER_TESTS_FAILED',
+          '[] Failed to trigger tests: query on baseURLurl returned: "Bad Gateway"\n'
```

Comment:
I need to fix this "baseURLurl". This issue comes from `formatBackendErrors()`, which directly concatenates the `baseURL` with the `url` from the Axios error. Axios [handles this internally](https://github.com/axios/axios/blob/1f13dd7e26124a27c373c83eff0a8614acc1a04f/lib/core/Axios.js#L123) and we may want to do the same as they do (cf. their [test file](https://github.com/axios/axios/blob/de5e006142cb428e712d00c05bda17c3c4c83ef1/test/specs/core/buildFullPath.spec.js)).